apps/hpe-csi/data.yaml

```diff
@@ -0,0 +1,39 @@
+tags:
+  - Storage
+  - Drivers and plugins
+title: "HPE CSI Driver"
```

Comment:
The name approved by HPE legal is "HPE CSI Driver for Kubernetes".

Reply:
No strict limit. E.g. here: https://github.com/k0rdent/catalog/blob/main/apps/dapr/data.yaml I can also check other apps in Catalog.